client/src/pages/Explore.js

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import { Typography } from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
+import moment from 'moment';
+
+import * as ExploreStyles from '../styles/ExploreSyles';
+import ProjectCard from '../components/ProjectCard';
+import Loading from '../components/Loading';
+import { withPageContext } from '../components/pageContext';
+import { OutlinedSelect } from '../components/Inputs';
+import { getUserProjects } from '../api/projects';
+import ProjectGrid from '../components/ProjectGrid';
+
+class Explore extends React.Component {
```

Comment:
When I open the page without being logged in, it just loads forever. Could use some sort of conditional render for that.

Reply:
Ya i'll need to optimize, might be fetching from backend issue, thanks

Reply:
Pushed a fix, that just might have been calling the backend with userId as null